### PR TITLE
Rename Variable to BallSize

### PR DESCRIPTION
**Pull request by NicoleFaye**
Identified Var as ball size.

### CODE REVIEW

File: engine/pinball_game/draw_sprites/draw_pinball.asm [hunk 1]
```diff
@@ -46,7 +46,7 @@ DrawPinball: ; 0x17e81
 	ld hl, hSCY
 	sub [hl]
 	ld c, a
-	ld a, [wd4c7]
```
Comment (huderlem) [inline engine/pinball_game/draw_sprites/draw_pinball.asm]: In a couple places, `wd4c7` was mistakenly replaced with `wBallSize`.

**Review by huderlem — CHANGES_REQUESTED** [review]
The name `wBallSize` seems good to me.  However, you'll have to revert/fix the changes to `wd4c7`, since that results in an incorrect ROM.

**Comment by NicoleFaye** [comment]
my bad, I thought I undid that mistake, it should be fixed now.

File: engine/pinball_game/draw_sprites/draw_pinball.asm (hunk 1 above)
Comment (NicoleFaye) [reply engine/pinball_game/draw_sprites/draw_pinball.asm]: Adjusted

**Pull request merged**